.github/workflows/schema-upgrade-check.yml

```diff
@@ -0,0 +1,177 @@
+# workflows/schema-upgrade-check.yml
+#
+# Determine if a commit introduces an extension schema change
+
+name: Check for Extension Schema Change
+
+on:
+  pull_request:
+    types: [opened, synchronize, reopened, ready_for_review]
+    branches:
+      - dev
+      - main
+    paths:
+      - ".github/workflows/schema-upgrade-check.yml"
+      - "pg_search/**"
+      - "!pg_search/README.md"
+  push:
+    branches:
+      - dev # Run CI on dev. This is important to fill the GitHub Actions cache in a way that pull requests can see it
+
+# because we may post a comment to the PR
+permissions:
+  pull-requests: write
+
+concurrency:
+  group: schema-upgrade-check-pg_search-${{ github.head_ref || github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  schema-upgrade-check:
+    name: Does this commit introduce a schema change?
+    runs-on: ${{ matrix.runner }}
+    if: github.event.pull_request.draft == false
+    strategy:
+      matrix:
+        include:
+          - runner: depot-ubuntu-latest-8
+            pg_version: 13
+            arch: amd64
+
+    env:
+      default_pg_version: 13
```

Comment:
Is this a limitation of `pg-schema-diff`? Heads up we don't need this whole `dev` skipping machinery in here since this doesn't need to run on `dev`

Reply:
Yes. Also 13 is out LCD anyways. Or will be come November when 12 EOL’s